### PR TITLE
feat: add ecosystem and eligible activity tags to credit class page

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1722

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

https://deploy-preview-1965--regen-registry.netlify.app/credit-classes/C02

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie you can test this on https://deploy-preview-1965--regen-registry.netlify.app/credit-classes/C02

**Review by ryanchristo — APPROVED** [review]
Code and preview look great to me.

File: web-registry/src/hooks/useTags.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { getAllEcosystemQuery } from 'lib/queries/react-query/sanity/getAllEcosystemQuery/getAllEcosystemQuery';
+
+import { getIconsMapping } from 'components/organisms/ProjectTopSection/ProjectTopSection.utils';
+
+type Params = {
+  activities: string[] | undefined;
+  ecosystemTypes: string[] | undefined;
+};
+
+export const useTags = ({ activities, ecosystemTypes }: Params) => {
```
Comment (flagrede) [inline web-registry/src/hooks/useTags.ts:16]: nice refactor 👍

**Review by flagrede — APPROVED** [review]
Looks good !

**Comment by erikalogie** [comment]
LGTM!

**Pull request merged**